rustest-testing/src/sequential.rs

```diff
@@ -0,0 +1,104 @@
+//! Example showing how to write sequentially executed tests.
+//! `sleep` is used to introduce artificial delay and the unix timestamp is printed to show that
+//! they indeed run sequentially with no overlap in execution time.
+
+use std::thread::sleep;
+use std::time::Duration;
+
+pub struct Client {}
+
+impl Client {
+    pub fn new() -> Self {
+        println!("Hello, world!");
+        Self {}
+    }
+    pub fn add(&self, left: u64, right: u64) -> u64 {
+        sleep(Duration::from_millis(100));
+        left + right
+    }
```

Comment:
Clippy complains that new and add are never used.
Maybe define them under `#[cfg(test)]` or export `Client` in lib.rs to avoid the warning.